### DISCUSSION TITLE
[Bug]: How to find the charIndex and style of the textbox where the mouse is located in a deeply nested group?

### DISCUSSION THREAD

**Discussion by aswind7** [fabricjs/fabric.js#10275] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.3.0

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

None

### Link To Reproduction

https://codesandbox.io/p/sandbox/js8d45

### Steps To Reproduce
1.  visit the Reproduction link
2.  mouse move on the blue text, see the console print, it works
3. mouse move on the redtext, see the console print, it doesn't work.    o(╥﹏╥)o

### Expected Behavior

I want to get the index and style of a textbox in a deeply nested group. The normal text index and style can be obtained normally (the one with blue text)
![image](https://github.com/user-attachments/assets/8659a6e9-4424-46c4-86e7-d7a6c54697d9)

### Actual Behavior

This code doesn't seem to be triggered, is there something wrong with my implementation?
```
    event.target.getObjects().forEach((group) => {
      fabric.util.addTransformToObject(group, group.group.calcOwnMatrix());

      group.getObjects().forEach((el) => {
        fabric.util.addTransformToObject(el, el.group.calcOwnMatrix());
        if (el.type === "textbox") {
          textboxs.push(el);

          if (el.containsPoint(event.absolutePointer, undefined, true)) {
            const charIndex = target.getSelectionStartFromPointer(event);
            const style = target.getSelectionStyles(charIndex, charIndex + 1);
            console.log(`Character Index in group: ${charIndex}`, style);
          }
        }
      });
    });
```

**Comment by asturur** [comment]
Why you felt like this is a bug report and not a discussion in Q&A?
